assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -480,6 +500,9 @@ def main():
         azureml_pkg_denylist_logging_patterns=LOGS_TO_BE_FILTERED_IN_APPINSIGHTS,
     )
 
+    # Adding flavor map to args
+    args.__setattr__("flavor_map", FLAVOR_MAP)
```

Comment:
nit: we can use `setattr(args, "flavor_map", FLAVOR_MAP)`

Reply:
Thanks, Fixed